wordpress-mobile/WordPress-iOS
# Add editor screen for unified prologue carousel

Ref #15056 

To test:

- Build, run and log out if necessary
- Verify that the editor (second) page of the login prologue is consistent with the designs (see issue #15056; see also example screenshot below)

<p align=center>
<img height=500 src=https://user-images.githubusercontent.com/34376330/112550321-2f653880-8d8d-11eb-9bc4-dca51209a3e6.png>
</p>

Known issues (to be fixed in the next PR):
- Background with colored circles is not there yet

PR submission checklist:
